Makefile

```diff
@@ -106,6 +106,19 @@ test-all:
 	@echo "Running all tests (including slow and online tests) ..."
 	@bash -c 'pytest -v tests/'
 
+.PHONY: test-notebooks
+test-notebooks:
+	@echo "Running notebook-based tests"
+	@bash -c "curl -L https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/raw/master/notebooks/output-sanitize.cfg --output $(CURDIR)/docs/source/output_sanitize.cfg --silent"
+	@bash -c "env FLYINGPIGEON_WPS_URL=$(FLYINGPIGEON_WPS_URL) pytest --nbval --verbose $(CURDIR)/docs/source/notebooks/ --sanitize-with $(CURDIR)/docs/source/output_sanitize.cfg --ignore $(CURDIR)/docs/source/notebooks/.ipynb_checkpoints"
```

Comment:
`FLYINGPIGEON_WPS_URL` is not defined.  Also please just name it `WPS_URL` so that `test-notebooks` target can be used as-is across all birds.  I intend to refactor the Makefile into re-usable generic pieces so refreshing the cookie-cutter do not generate so many merge conflicts.

Reply:
@tlvu the `FLYINGPIGEON_WPS_URL` differs on each server. How should it be defined?

Reply:
@nilshempelmann define it for the one on `localhost` !  Same target can the be used on Travis-CI which launch a local server, see https://github.com/bird-house/finch/blob/4e9d2baaad0166a76c3e49481934c1eb56af007a/Makefile#L4-L5

Reply:
@tlvu the server `url` is defined in `etc/custom.cfg` or as `localhost:5000` per default. the  `make test-notebooks` should read out `etc/custom.cfg`.